src/over.js

```diff
@@ -3,7 +3,8 @@ var _curry3 = require('./internal/_curry3');
 
 /**
  * Returns the result of "setting" the portion of the given data structure
- * focused by the given lens to the given value.
+ * focused by the given lens to the result of applying the given function to
+ * the focused value.
```

Comment:
is there a way to include more prepositions in that sentence? :stuck_out_tongue: